widget/button.go

```diff
@@ -13,13 +13,15 @@ import (
 )
 
 type Button struct {
-	Image             *ButtonImage
-	KeepPressedOnExit bool
-	ToggleMode        bool
-	GraphicImage      *ButtonImageImage
-	TextColor         *ButtonTextColor
-
-	//Allows the user to disable space bar and enter automatically triggering a focused button.
+	Image                   *ButtonImage
+	mask                    []byte
```

Comment:
Small request, Please move this to below "graphic *Graphic" on line 45

Reply:
done